kingpin/actors/rightscale/alerts.py

```diff
@@ -0,0 +1,358 @@
+              to.
+
+        Return:
+            [<rightcale.Resource objects>]
+        """
+        log.debug('Searching for AlertSpec matching: %s' % name)
+        found_spec = yield self._client.find_by_name_and_keys(
+            self._client._client.alert_specs, name, exact=False,
+            subject_href=subject_href)
+
+        if not found_spec:
+            log.debug('AlertSpec matching "%s" could not be found.' % name)
+            return
+
+        log.debug('Got AlertSpec: %s' % found_spec)
+        raise gen.Return(found_spec)
+
+
+class Create(AlertsBaseActor):
+
+    """Create a RightScale Alert Spec
+
+    **Options**
+
+    :array:
+      The name of the Server or ServerArray to create the AlertSpec on.
+
+    :strict_array:
+      Whether or not to fail if the Server/ServerArray does not exist.
+      (default: False)
+
+    :condition:
+      The condition (operator) in the condition sentence.
+      (`>, >=, <, <=, ==, !=`)
+
+    :description:
+      The description of the AlertSpec.
+      (*optional*)
+
+    :duration:
+      The duration in minutes of the condition sentence.
+      (`^\d+$`)
+
+    :escalation_name:
+      Escalate to the named alert escalation when the alert is triggered.
+      (*optional*)
+
+    :file:
+      The RRD path/file_name of the condition sentence.
+
+    :name:
+      The name of the AlertSpec.
+
+    :threshold:
+      The threshold of the condition sentence.
+
+    :variable:
+      The RRD variable of the condition sentence
+
+    :vote_tag:
+      Should correspond to a vote tag on a ServerArray if vote to grow or
+      shrink.
+
+    :vote_type:
+      Vote to grow or shrink a ServerArray when the alert is triggered. Must
+      either escalate or vote.
+      (`grow` or `shrink`)
+
+    **Examples**
+
+    Create a high network activity alert on my-array:
+
+    .. code-block:: json
+
+       { "desc": "Create high network rx alert",
+         "actor": "rightscale.alerts.Create",
+         "options": {
+           "array": "my-array",
+           "strict_array": true,
+           "condition": ">",
+           "description": "Alert if amount of network data received is high",
+           "duration": 180,
+           "escalation_name": "Email Engineering",
+           "file": "interface/if_octets-eth0",
+           "name": "high network rx activity",
+           "threshold": "50000000",
+           "variable": "rx"
+         }
+       }
+
+    **Dry Mode**
+
+    In Dry mode this actor *does* validate that the ``array`` array exists.
+    If it does not, a `kingpin.actors.rightscale.api.ServerArrayException` is
+    thrown. Once that has been validated, the dry mode execution simply logs
+    the Alert Spec that it would have created.
+
+    Example *dry* output::
+
+        TODO: Fill this in
+    """
+
+    all_options = {
+        'array': (str, REQUIRED, 'Name of the ServerArray act on.'),
+        'strict_array': (bool, False, 'Strict ServerArray validation.'),
```

Comment:
This description doesn't match the Sphinx description above.

Also what's the behavior if the array is not found?